Build/index.md

```diff
@@ -27,133 +34,19 @@ make
 make test
 ```
 
-## Prerequisites
-Development for NJOY uses the latest published language standards that are widely supported by compiler vendors, *at the start of development*. Therefore, the minimum language standards are
-
- - C++14 or higher
- - Fortran 2003 or higher
- - [Python 3.4+](https://python.org/)
-
-Additionally, we use [CMake](https://cmake.org/) to configure the build system and [git](https://git-scm.com) for version control.
-
- - [CMake 3.2+](https://cmake.org/) 
- - [git 2.1+](https://git-scm.com)
-
-
-## Build Process
-
-### Downloading
-To download NJOY2016, simply `git clone` the repository. First move into the directory where you want the source code
-
-```bash
-git clone https://github.com/njoy/NJOY2016.git
-```
-Similarly for NJOY21
-
-```bash
-git clone https://github.com/njoy/NJOY21.git
-```
-
-#### Updating NJOY to Incorporate Changes
-If you already have NJOY cloned then you can easily update to the latest version with a simple command:
-
-```bash
-git pull
-```
-This will "pull" all the changes that have been made to your local machine. You can continue to follow the instructions for configuring and building.
-
-    
-### Configuring
-The configuration and build is performed in a directory (`bin`) inside the source directory
-
-```bash
-cd NJOY21
-mkdir bin
-cd bin
-cmake ../
-```
-
-Note this will require a connection to the internet as `cmake` command will download (`clone`) the necessary dependencies. These will be placed in the `dependencies` directory.
-
-### Compiling/Building
-
-```bash
-make
-```
-You can provide the command-line option `-j n` (where `n` is the number of concurrent/parallel "jobs") to speed up the compilation step.
-
-### Testing
-
-
-```bash
-make test
-```
-Like for the compiling step, you can pass the `-j` command-line option here as well.
-
-If all the tests pass, you should see something that looks like:
-
-```
-100% tests passed, 0 tests failed out of 90
-```
-
-### Building Offline
-For this configuration/build process, a connection to the internet is required as subprojects are are downloaded from GitHub. There are many instances when one would need to build NJOY on a machine that is not connected to the internet. To do that, please use these steps:
-
+#### Option B (Docker):
```

Comment:
I'm not fully ready to support Docker at this moment. I'm not sure that including the `Dockerfile` in the NJOY2016 repository is the correct place to keep it. Until we decide exactly where we will permanently keep it, we shouldn't include the instructions here.